managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedgerFactory.java

```diff
@@ -179,4 +180,22 @@ void asyncOpenReadOnlyCursor(String managedLedgerName, Position startPosition, M
      */
     CompletableFuture<Boolean> asyncExists(String ledgerName);
 
+    /**
+     * @return return EntryCacheManager.
+     */
+    EntryCacheManager getEntryCacheManager();
+
+    /**
+     * update cache size and evictionTriggerThreshold.
+     *
+     * @param maxSize max cache size.
+     */
+    void updateCacheSizeAndThreshold(long maxSize);
```

Comment:
It's better to update `cacheSize` and `threahold` directly to `EntryCacheManager` to keep consistence to `managedLedgerCacheEvictionWatermark`

Reply:
Fixed